cnames_active.js

```diff
@@ -245,6 +245,7 @@ var cnames_active = {
   "auto-task-doc": "cname.vercel-dns.com", // noCF
   "autodocs": "bguiz.github.io/autodocs", // noCF? (don´t add this in a new PR)
   "automic": "automicjs.github.io",
+  "automata": "cname.vercel-dns.com", // noCF
```

Comment:
```suggestion
  "automata": "cname.vercel-dns.com", // noCF
  "automic": "automicjs.github.io",
```
As the CI error notes, your entry needs to be in alphabetical order

Reply:
Edited it, should be okay now.